rbuf.go

```diff
@@ -226,6 +226,19 @@ func (b *FixedSizeRingBuf) Write(p []byte) (n int, err error) {
 	}
 }
 
+// When there are no space to write, advance some space for write
```

Comment:
The go tools like vet will complain if the doc comment does not start with the name of the function.